guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -1,133 +1,52 @@
 [id="upgrading_{smart-proxy-context}_server_{context}"]
 = Upgrading {SmartProxyServers}
 
-ifdef::katello,orcharhino,satellite[]
+ifdef::satellite[]
 After you have upgraded your {ProjectServer} and synchronized the required repositories, you can start upgrading your {SmartProxyServers}.
 endif::[]
-ifndef::katello,orcharhino,satellite[]
+ifndef::satellite[]
 After you have upgraded your {ProjectServer}, you can start upgrading your {SmartProxyServers}.
 endif::[]
 
-.Prerequisites
-ifdef::katello,orcharhino,satellite[]
-* If you use content views to control updates to the {SmartProxyServer} base operating system or repository, update those content views with new repositories, then publish and promote their updated versions.
-For more information, see {ContentManagementDocURL}Managing_Content_Views_content-management[Managing content views] in _{ContentManagementDocTitle}_.
-* Ensure the base system of the {SmartProxy} is registered to the newly upgraded {ProjectServer}.
-endif::[]
-* If you implemented custom certificates, you must retain the content of both the `/root/ssl-build` directory and the directory in which you created any source files associated with your custom certificates.
-ifdef::katello,orcharhino,satellite[]
-+
-[WARNING]
+ifndef::foreman-deb[]
+[NOTE]
 ====
-Failure to retain these files during an upgrade causes the upgrade to fail.
-If these files have been deleted, they must be restored from a backup in order for the upgrade to proceed.
+You can use the *{SmartProxy} Upgrade Playbook* job template to upgrade {SmartProxyServer}s remotely.
+For more information about running remote jobs based on templates, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and setting up remote jobs] in _{ManagingHostsDocTitle}_.
 ====
 endif::[]
 
-.Upgrading {SmartProxyServers}
-. Create a backup.
-+
-* On a virtual machine, take a snapshot.
-* On a physical machine, create a backup.
-+
-For information on backups, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
-
-ifdef::foreman-el,katello[]
-. Update repositories:
-+
-[options="nowrap" subs="attributes"]
-----
-ifdef::katello[]
-# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm \
-https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/x86_64/katello-repos-latest.rpm
+.Prerequisites
+* The base system of the {SmartProxy} is registered to the newly upgraded {ProjectServer}.
+ifdef::katello,orcharhino,satellite[]
```

Comment:
Why are these conditions so complicated? It makes them difficult to understand.
I'd drop this and...